src/content/blog/ddev-feb-2025-newsletter.md

```diff
@@ -27,16 +27,16 @@ categories:
 - Tag1 Consulting did a performance comparison of the new Drupal CMS vs WordPress, (using DDEV for everything). See [Drupal CMS vs WordPress Performance](https://www.tag1consulting.com/blog/drupal-cms-vs-wordpress-performance-2025).
-- The latest DDEV Quickstarts in the docs are for [Drupal CMS](https://ddev.readthedocs.io/en/stable/users/quickstart/#drupal-drupal-cms) and [ProcessWire](https://ddev.readthedocs.io/en/latest/users/quickstart/#processwire)! Congratulations to the Drupal community on Drupal CMS, and welcome ProcessWire folks. Thanks to [Bernhard Baumrock](https://github.com/BernhardBaumrock) for the ProcessWire initiative!
+- The latest DDEV Quickstarts in the docs are for [Drupal CMS](https://docs.ddev.com/en/stable/users/quickstart/#drupal-drupal-cms) and [ProcessWire](https://docs.ddev.com/en/latest/users/quickstart/#processwire)! Congratulations to the Drupal community on Drupal CMS, and welcome ProcessWire folks. Thanks to [Bernhard Baumrock](https://github.com/BernhardBaumrock) for the ProcessWire initiative!
 - [DDEV: We use it on all our projects](https://www.youtube.com/watch?v=WkOoPEbtHwI). (Followups in [Reddit r/php](https://www.reddit.com/r/PHP/comments/1ijsev2/ddev_we_use_it_on_all_our_projects/) and [Reddit r/drupal](https://www.reddit.com/r/drupal/comments/1ijsemd/ddev_we_use_it_on_all_our_projects/)).
 
 **Ongoing DDEV Work**
 
-- **Node.js** improved support: People have asked for some time for Node.js to be the primary web server (instead of Nginx+php-fpm, etc), and now it can be done in a number of ways. This is now in [DDEV HEAD](https://ddev.readthedocs.io/en/stable/developers/building-contributing/#testing-latest-commits-on-head) and there are Quickstarts for [Simple Node.js](https://ddev.readthedocs.io/en/latest/users/quickstart/#nodejs-nodejs-web-server) and [SvelteKit](https://ddev.readthedocs.io/en/latest/users/quickstart/#nodejs-sveltekit). Incidentally, this also provides experimental support for [FrankenPHP](https://frankenphp.dev/) and provides a [Quickstart](https://ddev.readthedocs.io/en/latest/users/quickstart/#generic-frankenphp) for it as a demonstration of the "generic" web server type.
+- **Node.js** improved support: People have asked for some time for Node.js to be the primary web server (instead of Nginx+php-fpm, etc), and now it can be done in a number of ways. This is now in [DDEV HEAD](https://docs.ddev.com/en/stable/developers/building-contributing/#testing-latest-commits-on-head) and there are Quickstarts for [Simple Node.js](https://docs.ddev.com/en/latest/users/quickstart/#nodejs-nodejs-web-server) and [SvelteKit](https://docs.ddev.com/en/latest/users/quickstart/#nodejs-sveltekit). Incidentally, this also provides experimental support for [FrankenPHP](https://frankenphp.dev/) and provides a [Quickstart](https://docs.ddev.com/en/latest/users/quickstart/#generic-frankenphp) for it as a demonstration of the "generic" web server type.
```

Comment:
```suggestion
- **Node.js** improved support: People have asked for some time for Node.js to be the primary web server (instead of Nginx+php-fpm, etc), and now it can be done in a number of ways. This is now in [DDEV HEAD](https://docs.ddev.com/en/stable/developers/building-contributing/#testing-latest-commits-on-head) and there are Quickstarts for [Simple Node.js](https://docs.ddev.com/en/stable/users/quickstart/#nodejs-nodejs-web-server) and [SvelteKit](https://docs.ddev.com/en/stable/users/quickstart/#nodejs-sveltekit). Incidentally, this also provides experimental support for [FrankenPHP](https://frankenphp.dev/) and provides a [Quickstart](https://docs.ddev.com/en/stable/users/quickstart/#generic-frankenphp) for it as a demonstration of the "generic" web server type.
```